_posts/2022-04-25-tardigrade-launch.md

```diff
@@ -0,0 +1,301 @@
+---
+layout: post
+title:  "ETL in Trino speeds with project Tardigrade: now ready for early users"
+author: Andrii Rosa, Brian Zhan, Lukasz Osipiuk, Zebing Lin
+excerpt_separator: <!--more-->
+---
+
+After 6 months of challenging work, we’ve launched Project Tardigrade. It is 
+designed to improve user experience of running resource intensive queries that
+are common in ETL space. It’s available right now! It required some significant
+and fascinating engineering to get us here. Read on to learn how Project 
+Tardigrade works and how to enable it. We think you’ll enjoy hearing about it.
+
+<!--more-->
+
+## What is Project Tardigrade
+
+What we love most about doing ETL with Trino is that you get fast query speeds,
```

Comment:
ETL is an overloaded term. We should describe what kind of workloads we're talking about. In fact, what we're referring to as ETL here differs from the traditional meaning of ETL. See https://en.wikipedia.org/wiki/Extract,_transform,_load

Reply:
Rephrased to

> What we love most about Trino ...